### PR TITLE
add NET_RAW capability to server container in "should drop INVALID conntrack entries" e2e test

### PR DESCRIPTION
**Pull request by ricky-rav**
The boom-server container forges out-of-order TCP packets and injects them into the network. This requires the container to have the CAP_NET_RAW linux capability, otherwise the test will fail.

For security reasons, NET_RAW capability was removed as default capabilities in OpenShift Container Platform 4.6, causing "should drop INVALID conntrack entries" e2e test to fail: https://bugzilla.redhat.com/show_bug.cgi?id=1997235

/kind failing-test
/kind regression

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/network/conntrack.go [hunk 1]
```diff
@@ -325,6 +325,11 @@ var _ = common.SIGDescribe("Conntrack", func() {
 								},
 							},
 						},
+						SecurityContext: &v1.SecurityContext{
+							Capabilities: &v1.Capabilities{
+								Add: []v1.Capability{"NET_RAW"},
+							},
```
Comment (aojea) [inline test/e2e/network/conntrack.go:331]: Add the following tag `[Privileged]` to the test title please
```
	ginkgo.It("should drop INVALID conntrack entries [Privileged]", func() {
```

Comment (ricky-rav) [reply test/e2e/network/conntrack.go:331]: Done!

**Comment by aojea** [comment]
/lgtm
/approve
Thanks

**Pull request merged**